presto-main/src/main/java/com/facebook/presto/sql/planner/QueryPlanner.java

```diff
@@ -566,14 +565,6 @@ private PlanBuilder aggregate(PlanBuilder subPlan, QuerySpecification node)
                 Optional.empty(),
                 groupIdSymbol);
 
-        if (aggregationNode.hasEmptyGroupingSet() && aggregationNode.hasNonEmptyGroupingSet() && aggregationNode.hasOrderings()) {
-            // Having both empty grouping set and non-empty grouping set will require partial aggregation.
```

Comment:
``` empty grouping set and non-empty grouping set will require partial aggregation```
This is still true, right?

```Since aggregation with ORDER BY does not support partial aggregation, we can not allow queries to have both.```
Isn't this still true?

I guess by https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AggregationNode.java#L210 such aggregation will be executed just on single node single thread....

Reply:
No, not needed anymore. As you guessed, they will run on a single node. Not ideal, but at least it's possible to do it now.

Reply:
Right. This statement is false:
```
// Having both empty grouping set and non-empty grouping set will require partial aggregation.
```